app/src/main/java/nl/tudelft/b_b_w/view/FriendsPageActivity.java

```diff
@@ -15,39 +26,127 @@
 public class FriendsPageActivity extends Activity {
 
```

Comment:
I think you should change the old ratings (5 stars bar) into the new one that is created in the contacts list. The amount of stars should depend on the trust values of the person you paired with.